mava/configs/env/connector.yaml

```diff
@@ -10,6 +10,9 @@ env_name: MaConnector # Used for logging purposes.
 # This metric is returned at the end of an experiment and can be used for hyperparameter tuning.
 eval_metric: episode_return
 
+# Whether the environment observations will be a feature vector or 2-dimensional
+observation_type: vector # [pixel, vector]
```

Comment:
Note: I am not sure that `pixel` is the correct word here. Perhaps `grid_based` is better? But keen to hear what people think.